### PR TITLE
Cool Coin Registration pull request

### PR DESCRIPTION
**Pull request by yothatscool**
pull request for new token registry.

### CODE REVIEW

File: tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/additional.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "website":"https://www.twitter.com/yothatscooldeux", 
+  "links":{
+      "twitter":"https://twitter.com/yothatscooldeux",
```
Comment (libotony) [inline tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/additional.json:4]: Please keep this and remove other examples.

**Review by libotony — CHANGES_REQUESTED** [review]

File: tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/additional.json (hunk 1 above)
Comment (yothatscool) [reply tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/additional.json]: Thank you for checking this. I have removed the examples and kept the necessary information.

File: tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/additional.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
```
[the remark this answers is not in the document]
Comment (yothatscool) [reply tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/additional.json]: of course... updated. Thank you!

**Comment by libotony** [comment]
The image should be 256px * 256px

File: tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/info.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+        "name": "COOL",
```
Comment (libotony) [inline tokens/main/0x2f2220139e46bcc98273ecca2ded7bf56373b6cf/info.json:2]: Message from the CI: name does not match contract name (info=COOL, contract=cool)

**Review by libotony — CHANGES_REQUESTED** [review]

**Review by libotony — APPROVED** [review]

**Pull request merged**